modules/hcp-configuring-oauth-console.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-authentication-authorization.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-configuring-oauth-console_{context}"]
+= Configuring the OAuth server for a hosted cluster by using the web console
+
+You can configure the internal OAuth server for your hosted cluster by using the {product-title} web console.
+
+You can configure OAuth for the following supported identity providers:
+
+* `oidc`
+* `htpasswd`
+* `keystone`
+* `ldap`
+* `basic-authentication`
+* `request-header`
+* `github`
+* `gitlab`
+* `google`
+
+Adding any identity provider in the OAuth configuration removes the default `kubeadmin` user provider.
+
+.Prerequisites
+
+* You logged in as a user with `cluster-admin` privileges.
+* You created your hosted cluster.
+
+.Procedure
+
+. Navigate to *Home* -> *API Explorer*.
+
+. Use the *Filter by kind* box to search for your `HostedCluster` resource.
+
+. Click the `HostedCluster` resource that you want to edit.
+
+. Click the *Instances* tab.
+
+. Click the Options menu {kebab} next to your hosted cluster name entry and click *Edit HostedCluster*.
+
+. Add the OAuth configuration in the YAML file:
++
+[source,yaml]
+----
+spec:
+  configuration:
+    oauth:
+      identityProviders:
+      - openID: <1>
+          claims:
+            email: <2>
+              - <email_address>
+            name: <3>
+              - <display_name>
+            preferredUsername:
+              - <preferred_username> <4>
```

Comment:
Honestly this is not worth holding up merging over so I won't but just something that you can consider fixing in a separate PR if you'd like - I think based on the description of this callout and the way you use callouts 1-3, maybe this callout should go on the line above this one instead.

Reply:
@skopacz1 Good catch! Opened a PR to fix this in both  - CLI and web console sections.
https://github.com/openshift/openshift-docs/pull/75647